Docs/source/usage/parameters.rst

```diff
@@ -1259,6 +1259,20 @@ Particle initialization
     Resampling is performed everytime the number of macroparticles per cell of the species
     averaged over the whole simulation domain exceeds this parameter.
 
+* ``<species>.do_supercycling`` (`0` or `1`) optional (default `0`)
```

Comment:
This flag is not really needed as a separate input argument. Giving `supercycling_interval` a value other than 1 would be sufficient to turn on the super cycling.